src/mmw/apps/modeling/views.py

```diff
@@ -229,6 +242,9 @@ def _initiate_tr55_job_chain(model_input, job_id):
 
 
 def _construct_tr55_job_chain(model_input, job_id):
+    exchange = 'C.dq'
```

Comment:
Yes, this is a special magic exchange name that is created by Celery when you ask it to create per-worker queues (by setting `CELERY_WORKER_DIRECT = True`).